### DISCUSSION TITLE
Integration on point-spread-functions (PSF)

### DISCUSSION THREAD

**Discussion by Constantin-Jehn** [Project-MONAI/MONAI#4177] [Q&A]
Hello,

I am relatively new to monai and I try to build slice to volume reconstruction from motion corrupted MRI data.
For updating the volume from 2d slice one would usually use PSFs like a Gaussian or sinc functions.

As far as I understand monai (as well as other libraries like simpleITK) are limited to modes in Resampling (nearest, bilinear etc.).

Has anyone experience incorporating more complex PSFs into monai?

Any thoughts, ideas or remarks are highly appreciated.

**Comment by wyli** [comment]
this work uses MONAI for the segmentation part -- https://github.com/gift-surg/NiftyMIC, but I think specific functions you are interested in are in [a customized version of itk](https://github.com/gift-surg/ITK_NiftyMIC/wiki)...(cc @renbem)
